### PR TITLE
feat: add md breackpoint to cta-block

### PR DESCRIPTION
**Pull request by niktverd**

### CODE REVIEW

File: src/blocks/CTA/CTA.scss [hunk 1]
```diff
@@ -19,56 +19,91 @@ $block: '.#{$namespace}cta';
...
         display: flex;
-        padding-bottom: $indentXS;
+        padding-bottom: 0px;
+
+        &_layout {
+            width: 100%;
+            max-width: 100%;
+            margin-right: 0px;
+
+            &_2 {
```
Comment (yuberdysheva) [inline src/blocks/CTA/CTA.scss:29]: Can we set styles with nth-child?
isn't that strange that we named classes like numbers?

File: src/blocks/CTA/CTA.scss [hunk 2]
```diff
@@ -19,56 +19,91 @@ $block: '.#{$namespace}cta';
...
             &_layout {
-                width: 100%;
-                max-width: 100%;
-                margin-right: 0px;
-
-                &_2 {
+                &_1,
+                &_2,
+                &_3,
+                &_4 {
```
Comment (yuberdysheva) [inline src/blocks/CTA/CTA.scss:76]: can we set one common class for all four elements?

File: src/blocks/CTA/CTA.scss [hunk 3]
```diff
@@ -19,56 +19,91 @@ $block: '.#{$namespace}cta';
...
-
-                &_3 {
-                    width: calc((100% / 3) - (#{$indentM} / 3));
-                    $max-width: 50%;
-                    margin-right: $indentXS;
+                &_1:nth-child(2n),
+                &_2:nth-child(2n),
+                &_3:nth-child(2n),
+                &_4:nth-child(2n) {
+                    margin-right: 0;
```
Comment (yuberdysheva) [inline src/blocks/CTA/CTA.scss:87]: Is there a number and nth-children? In what case we use it?

File: src/blocks/CTA/CTA.scss [hunk 4]
```diff
@@ -19,56 +19,91 @@ $block: '.#{$namespace}cta';
...
-                    margin-right: 0px;
-                }
+        &__button {
+            padding-bottom: $indentXS;
 
+            &_layout {
+                &_1,
+                &_2,
+                &_3,
                 &_4 {
```
Comment (yuberdysheva) [inline src/blocks/CTA/CTA.scss:104]: can we set one common class for all four elements?

File: src/blocks/CTA/CTA.scss (hunk 1 above)
Comment (niktverd) [reply src/blocks/CTA/CTA.scss]: I got rid of such classes at all. Thank you for the comment

File: src/blocks/CTA/CTA.scss (hunk 2 above)
Comment (niktverd) [reply src/blocks/CTA/CTA.scss]: +

File: src/blocks/CTA/CTA.scss (hunk 3 above)
Comment (niktverd) [reply src/blocks/CTA/CTA.scss]: +

File: src/blocks/CTA/CTA.scss (hunk 4 above)
Comment (niktverd) [reply src/blocks/CTA/CTA.scss]: +

File: src/blocks/CTA/CTA.scss [hunk 5]
```diff
@@ -19,58 +19,43 @@ $block: '.#{$namespace}cta';
 
     &__button {
         display: flex;
-        padding-bottom: $indentXS;
+        padding-bottom: 0px;
```
Comment (yuberdysheva) [inline src/blocks/CTA/CTA.scss:22]: is it necessary zero here?

File: src/blocks/CTA/CTA.scss [hunk 6]
```diff
@@ -19,58 +19,43 @@ $block: '.#{$namespace}cta';
 
     &__button {
```
Comment (yuberdysheva) [inline src/blocks/CTA/CTA.scss:20]: I saw a layout, why the whole item's name is button? It's not a button
image

**Review by yuberdysheva — CHANGES_REQUESTED** [review]

File: src/blocks/CTA/CTA.scss (hunk 6 above)
Comment (niktverd) [reply src/blocks/CTA/CTA.scss:20]: agree

**Review by yuberdysheva — CHANGES_REQUESTED** [review]

File: src/blocks/CTA/CTA.scss (hunk 5 above)
Comment (niktverd) [reply src/blocks/CTA/CTA.scss]: Agree

File: src/blocks/CTA/CTA.scss [hunk 7]
```diff
@@ -17,60 +17,49 @@ $block: '.#{$namespace}cta';
         padding: $indentM;
     }
 
-    &__button {
+    &__layout {
```
[the remark this answers is not in the document]
Comment (niktverd) [reply src/blocks/CTA/CTA.scss]: fix

**Review by yuberdysheva — APPROVED** [review]

**Review by yuberdysheva — APPROVED** [review]

**Review by NikitaCG — APPROVED** [review]

**Pull request merged**